Source/FieldSolver/ElectrostaticSolver.cpp

```diff
@@ -346,8 +350,14 @@ WarpX::computePhiCartesian (const amrex::Vector<std::unique_ptr<amrex::MultiFab>
 
     // Solve the Poisson equation
     linop.setDomainBC( lobc, hibc );
+
+    // This is apparently necessary for how AMReX expects phi
+    for (int lev=0; lev < rho.size(); lev++){
+        phi[lev]->mult(-1.*PhysConst::ep0);
```

Comment:
Good catch adding this. Maybe it makes more sense now to multiply rho by -1/ep0? rho is a temporary so it wouldn't have to be unscaled after the solve.

Reply:
I agree scaling rho is cleaner. I made the corresponding change in both `computePhiCartesian` and `computePhiRZ`.